src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php

```diff
@@ -80,7 +85,7 @@ public function render(string|array $entryPoint, array $attributes = []): string
                 // importmap entry is a noop
                 $importMap[$importName] = 'data:application/javascript,';
             } else {
-                $importMap[$importName] = 'data:application/javascript,'.rawurlencode(\sprintf('document.head.appendChild(Object.assign(document.createElement("link"),{rel:"stylesheet",href:"%s"}))', addslashes($path)));
```

Comment:
Shouldn't we keep `rawurlencode`?

Reply:
I replaced it by a str_replace('%', '%25', ...)`, that's enough, browsers don't require more chars to be encoded and that improves both readability and payload size.